SixTest/CheckTestInputForBOINC.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+for i in $(ls -d */); do
+#echo ${i%%/};
+if [ -f ${i}Sixin.zip ]; then
+	mkdir ${i}/zip
+	cp ${i}/Sixin.zip ${i}/zip
+	cd ${i}/zip
+	unzip -qq Sixin.zip
+	for j in $(ls fort.*); do
```

Comment:
see for example `javier_bignpart` (no Sixin? huh.), `dynk_globalvars`, or `thick6ddynk`. They both have extra_inputs.txt.